Source/WTF/wtf/cocoa/RuntimeApplicationChecksCocoa.h

```diff
@@ -153,6 +153,7 @@ namespace CocoaApplication {
 WTF_EXPORT_PRIVATE bool isAppleApplication();
 WTF_EXPORT_PRIVATE bool isAppleBooks();
 WTF_EXPORT_PRIVATE bool isWebkitTestRunner();
+WTF_EXPORT_PRIVATE bool shouldOSFaultLogForAppleApplication();
```

Comment:
Let's make this method name more specific since it's about WK1 logging.

I don't want to bike shed on the name (easy to rename later), so if you don't like the one below, just pick something else and land.

```suggestion
WTF_EXPORT_PRIVATE bool shouldOSFaultLogForAppleApplicationUsingWebKit1();
```